translations/en.i18n.json

```diff
@@ -3,7 +3,8 @@
     "audioPlayer": {
       "enableLoop": "Enable loop",
       "disableLoop": "Disable loop",
-      "mediaThumb": "Media thumbnail"
+      "mediaThumb": "Media thumbnail",
+      "pluginIcon": "Plugins"
```

Comment:
is there a ticket for translation of the new field ?

Reply:
Should be

Reply:
Should be